sdk/identity/identity-web/README.md

```diff
@@ -0,0 +1,193 @@
+## Azure Identity Plugin for Web Environments
+
+This package provides a new credential for the Azure Identity library for JavaScript ([`@azure/identity`](https://npmjs.com/package/@azure/identity)) called `WebRedirectCredential`, that simplifies the development experience of web servers deployed to the Azure cloud.
+
+The `WebRedirectCredential` uses the [Authorization Code Flow](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow) to authenticate server-side applications with browser user interaction.
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity-web) | [Samples](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity-web/samples-dev)
+
+## Getting started
+
+```javascript
+const { WebRedirectCredential } = require("@azure/identity-web");
+const { KeyClient } = require("@azure/keyvault-keys");
+
+// Within a server route, like `app.get("/azureLogin"`:
+const credential = new WebRedirectCredential(
+  tenantId,
+  clientId,
+  redirectUri
+);
+
+// This credential works well with a `state` property that ties
+// the resulting token with the source of the redirection:
+const state = session.username; // or ID
+
+// Make sure to redirect to this URI to trigger the Authentication Code Flow:
+const authorizeUrl = credential.getRedirectUri(scope, {
```

Comment:
(Based on feedback from @willmtemple  and @KarishmaGhiya )

Use something similar to this: https://github.com/Azure/azure-sdk-for-js/pull/21028/files#diff-1f63193b2eb9f68e6353d08afcd823edf160f469d3c70c37d6631f98e052ece6R324

The question is: Should our clients expose a scope? If so, it would be much easier for our users to discover the scopes needed to authenticate with different clients:

```ts
const client = new ServiceBusClient(serviceBusEndpoint, credential);
const scopes = client.scopes;
await credential.authenticate(scopes);
```

Reply:
Jose suggested - client.authenticationOptions = {scopes: []}